cephfs/cephfs.go

```diff
@@ -7,17 +7,26 @@ package cephfs
 #include <cephfs/libcephfs.h>
 */
 import "C"
-import "fmt"
-import "unsafe"
 
-//
+import (
+	"fmt"
+	log "github.com/sirupsen/logrus"
+	"math"
+	"syscall"
+	"unsafe"
+)
+
 type CephError int
 
 func (e CephError) Error() string {
-	return fmt.Sprintf("cephfs: ret=%d", e)
+	if e == 0 {
```

Comment:
i would use: 

pseudo code
```golang
if e != 0 {
return err 
}

Reply:
How would `return fmt.Sprintf("cephfs: no error given")` work for you?  This function is expected to return a string, if it is given a non-error return value, we should politely indicate that the returned value was not an error.

Reply:
https://github.com/ceph/go-ceph/pull/74/commits/71cbf66dca3f99d5804039914b2f136e69c13042#diff-42cb3398d9c86351575e35bbc99bfdb9R23